src/core/Microsoft.PowerToys.Settings.UI.Lib/KeyboardManagerSettings.cs

```diff
@@ -18,21 +14,15 @@ public class KeyboardManagerSettings : BasePTModuleSettings
         public KeyboardManagerSettings()
         {
             Properties = new KeyboardManagerProperties();
-            version = "1";
-            name = "_unset_";
+            Version = "1";

```

Comment:
We can use the Microsoft.PowerToys.Settings.UI.Lib.Utilities.Helper.GetProductVersion() for this one

Reply:
Lets do that in a different PR.  Every settings file does this.